resdk/resolwe.py

```diff
@@ -243,7 +243,7 @@ def _process_file_field(self, path):
 
     def run(self, slug=None, input={}, descriptor=None,  # pylint: disable=redefined-builtin
             descriptor_schema=None, collections=[],
-            data_name='', src=None, tools=None):
+            data_name='', src=None, tools=None, use_existing=False):
```

Comment:
If given, check that they match the returned object. We should find a better parameter name for `use_existing`.